dask_ml/model_selection/_incremental.py

```diff
@@ -33,35 +33,40 @@ def _partial_fit(model_and_meta, X, y, fit_params):
         A new dictionary with updated information.
     """
     with log_errors():
+        start = time()
         model, meta = model_and_meta
 
         if len(X):
             model = deepcopy(model)
             model.partial_fit(X, y, **(fit_params or {}))
 
         meta = dict(meta)
-        meta['time_step'] += 1
+        meta["partial_fit_calls"] += 1
+        meta["partial_fit_time"] = time() - start
```

Comment:
Is it normal to include profiling information like this as part of the computation?  Is this what sklearn does?

Reply:
`cv_results` of a GridSearchCV or `cross_val_score` records the training time / predict time for each CV batch / parameter combination.

Reply:
OK, thank you for the clarification.  Is there a standard convention to record this?  If so then are we following that convention?

Reply:
I've resolved this by collecting a list of `partial_fit_times` and `score_times`. I'll let the caller of `fit` (i.e., HyperbandCV) decide how to process them.